pywebpack/helpers.py

```diff
@@ -19,7 +19,11 @@
 
 def bundles_from_entry_point(group):
     """Load bundles from entry point group."""
-    return (ep.load() for ep in pkg_resources.iter_entry_points(group))
+    loaded_eps = [ep.load() for ep in pkg_resources.iter_entry_points(group)]
```

Comment:
This function should load the entry point inside the iterator, otherwise, the loading of the entry point is done immediately during the function call to ``bundles_from_entry_point``, instead of only when the iterator is being accessed.

Take e.g. this code:

```python
# Line A
project = WebpackBundleProject(
    # ...
    bundles=bundles_from_entry_point('webpack_bundles'),
)
# Line B
for b in projects.bundles:
    # ...
```

With this PR, the entry points are being loaded in Line A, whereas with the old version the entry points are not being loaded until Line B.